docs/madr/decisions/049-policy-on-namespace.md

```diff
@@ -0,0 +1,529 @@
+* Good, because all consumers are treated equally by default without favoring the local ones.
+* Bad, because if you want to keep this only to your local zone, you need to specify the name of the zone in the policy
+
+### Order when merging
+
+By design, several policies can select the same DPP.
+In that case, policies are ordered and their `to` (or `from`) lists are merged.
+Introducing namespace-scoped policies adds another steps to the policy comparison process:
+
+1. Is top-level `targetRef.kind` more specific (MeshServiceSubset > MeshService > MeshSubset > Mesh)?
+   When kinds are equal, go to the next step.
+2. Zone originated policy is more specific than global originated policy.
+   When policies originates at the same place, go to the next step.
+3. **[new step]** Policy in any custom namespace is more specific than any policy in the `kuma-system` namespace.
+   When both policies either from custom namespaces or from `kuma-system`, go to the next step.
+4. **[new step]** Policy from local zone is more specific than a policy from another zone. 
+   When policies have the same zone, go to the next step.
+5. **[new step]** Policy from local namespace is more specific than a policy from another namespace.
+   When namespaces are equal, go to the next step.
+6. The policy is more specific if it's name is lexicographically less than other policy's name ("aaa" < "bbb" so that "
+   aaa" policy is more specific)
+
+### Cross-resource references
+
+Kuma policies support cross-policy references.
+At this moment, it works only between MeshTimeout and MeshHTTPRoute, but there are plans to support it for other
+policies, i.e. [#6645](https://github.com/kumahq/kuma/issues/6645).
+Referencing a policy from another namespace or another zone should be done in a similar way MeshService is referenced.
+Fields `name/namespace` and `labels` are mutually exclusive.
+
+```yaml
+# reference an object with name/namespace in the storage
+targetRef:
+  kind: MeshHTTPRoute
+  name: orders-route
+  namespace: backend-ns
+---
+# reference MeshHTTPRoute that was synced from 'us-east-1' 
+targetRef:
+  kind: MeshHTTPRoute
+  labels:
+    kuma.io/display-name: orders-route
+    k8s.kuma.io/namespace: backend-ns
+    kuma.io/zone: us-east-1
+```
+
+As mentioned previously, applying a policy on the custom namespace affects requests to the pods in the custom namespace.
+This means, no matter where MeshHTTPRoute is created it can be a "producer" route and affect consumers' configurations.
+It's always justifiable to reference any MeshHTTPRoute from any MeshTimeout.
+
+### MeshGateway reference
+
+`MeshGateway` at this moment is not namespace-scoped. It can change in the future with
+the [MeshBuiltinGateway resource](https://github.com/kumahq/kuma/issues/10014).
+
+For policies in custom namespaces, we've decided that we would allow targeting `MeshGateway` in `MeshHTTPRoute` and `MeshTCPRoute`, and policies 
```

Comment:
targeting where? to? top level?

Reply:
updated to be more precise about uing MeshGateway in topLevel targetRef